wgengine/netstack/netstack.go

```diff
@@ -1909,3 +1910,23 @@ func (ns *Impl) ExpVar() expvar.Var {
 
 	return m
 }
+
+// windowsPingOutputIsSuccess reports whether the ping.exe output b contains a
+// success ping response for ip.
+//
+// See https://github.com/tailscale/tailscale/issues/13654
+//
+// TODO(bradfitz,nickkhyl): delete this and use the proper Windows APIs.
+func windowsPingOutputIsSuccess(ip netip.Addr, b []byte) bool {
+	// Look for a line that contains " <ip>: " and then three equal signs.
+	// This heuristic seems to match the ping.exe output in any language.
```

Comment:
If the round-trip time is less than 1ms, ping.exe uses `<` instead of `=` in the formatted output.
For example,
```
Reply from 10.0.0.1: bytes=32 time<1ms TTL=64
```

Reply:
Good catch. Fixed, with new test.